tests/test-team/setup/session.setup.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable unicorn/prefer-module */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
is this needed?

I generally follow the rule of them where I don't inline disable a rule, unless it's a super edge case. If the rule isn't valid or I disagree with it. Then I'd disable it globally or update the code to fix the linting issue. 

I think in the case of the ones you've disabled they might not be relevant any more? Apart from the `no-await-in-loop`.

Which is valid. You can update your code to promisify it like so;

```
const promises = sessionData.map((session) => client.create(session));

await Promise.all(promises);
```

This also has additional benefits of being slightly faster because it's "concurrent". Also if any of the promises fail then it'll throw an error. The alternative is `Promise.allSettled` which doesn't fail but you have to reason over the response.

Reply:
This is cool, got a bit stuck with the async/await stuff but this is nice and clean 🦖